app/src/pages/inside/profilePage/modals/deleteAccountFeedbackModal/deleteAccountFeedbackModal.scss

```diff
@@ -0,0 +1,51 @@
+/*!
+ * Copyright 2023 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.options {
+  padding: 16px 16px 0 16px;
+}
+
+.option {
+  margin-top: 4px;
+  height: 30px;
+  line-height: 30px;
+
+  &:first-child {
+    margin-top: 0;
+  }
+
+  label {
```

Comment:
Please don't modify the nested components internals styles indirectly.
It can lead to bugs once we've change the component structure.

Reply:
Fixed.